### PR TITLE
xbps-checkvers

### PR DESCRIPTION
**Pull request by Duncaen**

### CODE REVIEW

File: bin/xbps-checkvers/main.c [hunk 1]
```diff
@@ -200,9 +200,10 @@ show_usage(const char *prog)
 "  -I,--installed 		Check for outdated packages in rootdir, rather\n"
 "				than in the XBPS repositories.\n"
 "  -i,--ignore-conf-repos	Ignore repositories defined in xbps.d.\n"
+"  -m,--manual	Only process listed files.\n"
```
Comment (xtraeme) [inline bin/xbps-checkvers/main.c]: Please fix the indent here.

**Comment by xtraeme** [comment]
so that with -m we can now specify a list of templates as arguments?

**Comment by xtraeme** [comment]
Manual page needs to be updated ?

**Comment by xtraeme** [comment]
oh I see, you've combined -s,--show-all with -m.

Great

**Comment by xtraeme** [comment]
:+1:

File: bin/xbps-checkvers/main.c (hunk 1 above)
Comment (Duncaen) [reply bin/xbps-checkvers/main.c]: Fixed it while making the usage text match all the other xbps-* usage outputs.

**Pull request merged**